modules/view-unhealthy-clusters.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * configuration/cluster-decommissioning.adoc
+:_content-type: PROCEDURE
+[id="view-unhealthy-clusters_{context}"]
+= Viewing unhealthy clusters
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *System Configuration*.
+. In the *Cluster deletion* section, click *Clusters which have Sensor Status: Unhealthy* to go to the Clusters list page. This page is filtered to show unhealthy clusters that are subject to deletion and a timeframe for the deletion. 
++
+[NOTE]
+====
```

Comment:
Please check the formatting of the note. For some reason, it's rendered as a code snippet within the note.